src/App.js

```diff
@@ -1,23 +1,37 @@
-import logo from "./logo.svg";
+import React, { useState } from "react";
+// import logo from "./logo.svg";
 import "./App.css";
 import Header from "./Header";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import Assets from "./Header/Assets";
 import Home from "./Header/Home";
 import Employees from "./Header/Employees";
+import Login from "./Header/Login";
 import { settingInitialDb } from "./Utilities";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 
 function App() {
   settingInitialDb();
+  const [isAuthenticated, setIsAuthenticated] = useState(false);
 
   return (
     <Router>
       <Header />
       <Routes>
-        <Route exact path="/" element={<Home />} />
-        <Route exact path="/home" element={<Home />} />
-        <Route path="/assets" element={<Assets />} />
-        <Route path="/employees" element={<Employees />} />
+        {isAuthenticated ? (
+          <>
```

Comment:
put routes here itself and remove from above

Reply:
Tried doing... but did not work.